packages/core/src/auth/sso/clients.ts

```diff
@@ -199,27 +199,17 @@ export class SsoClient {
     }
 }
 
-function omitIfPresent<T extends Record<string, unknown>>(obj: T, ...keys: string[]): T {
-    const objCopy = { ...obj }
-    for (const key of keys) {
-        if (key in objCopy) {
-            ;(objCopy as any)[key] = '[omitted]'
-        }
-    }
-    return objCopy
-}
-
 function addLoggingMiddleware(client: SSOOIDCClient) {
     client.middlewareStack.add(
         (next, context) => args => {
             if (HttpRequest.isInstance(args.request)) {
                 const { hostname, path } = args.request
-                const input = omitIfPresent(
+                const input = partialClone(
                     // TODO: Fix
                     args.input as unknown as Record<string, unknown>,
-                    'clientSecret',
-                    'accessToken',
-                    'refreshToken'
+                    3,
```

Comment:
Replaced `omitIfPresent` with a more generic `partialClone` function. It has some nice features which are often needed.

Ideally we would just use node's `util.inspect()`, but
- don't know a good browser-compatible polyfill
- only produces strings, not "partial objects"